composer.json

```diff
@@ -10,7 +10,9 @@
         "ast"
     ],
     "require": {
-        "rector/rector-prefixed": "0.7.29"
+        "php": "^7.2",
+        "jawira/case-converter": "^1.1",
```

Comment:
This was removed since we weren't using it any more. We'll have to remove it here as well.

Reply:
Sure, go for it.

I think I messed rebase, that's why it's here